src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -30,8 +30,18 @@ class Workflow
     private $dispatcher;
     private $name;
 
-    public function __construct(Definition $definition, MarkingStoreInterface $markingStore = null, EventDispatcherInterface $dispatcher = null, $name = 'unnamed')
-    {
+    /**
+     * @param Definition                    $definition
+     * @param MarkingStoreInterface|null    $markingStore
+     * @param EventDispatcherInterface|null $dispatcher
+     * @param string                        $name
+     */
+    public function __construct(
+        Definition $definition,
+        MarkingStoreInterface $markingStore = null,
+        EventDispatcherInterface $dispatcher = null,
+        $name = 'unnamed'
+    ) {
```

Comment:
We are not using multi-line constructors and never add phpdoc when it does not add anything that cannot be deduced from the signature, which looks like it is the case here.